www/_posts/2020-06-29-cordova-android-9.0.0.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 9.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 9.0.0`!  This is one of Cordova's supported platforms for building Android mobile applications.
+
+* [cordova-android@9.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
```

Comment:
Remove "Release Highlights" heading here, it's duplicated below (above the actual release highlights)